src/gradle.ts

```diff
@@ -84,8 +84,13 @@ export function getTaskToPublish(
             reject(new Error(ERROR_MULTIPLE_PLUGIN));
           }
           task = "publishPlugins";
+        } else {
+          logger.debug(line.toString())
         }
       });
+      child.stderr.pipe(split()).on("data", (line: string) => {
+          logger.error(line.toString())
```

Comment:
Do we need `.toString()` here even though we know that `line` is instanceof `string`?